docs/src/gcode/coordinates.adoc

```diff
@@ -52,7 +52,7 @@ will move from the current position to the position where
 the machine coordinates of the three axes will be at zero. You can
 use this command if you have a fixed position for the tool change or if
 your machine has an automatic tool changer. You can also use this command
-to clear the work area and access the workpiece in the vise.
+to clear the work area and access the workpiece in the vice.
```

Comment:
"vise" is correct here, this patch is wrong.

https://www.merriam-webster.com/words-at-play/vise-and-vice-they-mostly-arent-the-same-thing

Reply:
Apologies. Corrected in [ba0e7f6](https://github.com/LinuxCNC/linuxcnc/pull/2550/commits/ba0e7f62db21574e13ba5ce9aacb657d3b90620b)